CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 [Unreleased]
 ### Changed
+* Added unit tests for utilFncs.py
+* [686](https://github.com/dbekaert/RAiDER/pull/686) - Linted the project with `ruff`.
```

Comment:
Change the changelog message to what you actually did. @royagrace 
Also make sure the commit number is the correct number.